Source/WebCore/page/NavigationHistoryEntry.h

```diff
@@ -64,7 +69,7 @@ class NavigationHistoryEntry final : public RefCounted<NavigationHistoryEntry>,
     const WTF::UUID m_key;
     const WTF::UUID m_id;
     uint64_t m_index;
-    JSC::JSValue m_state;
+    RefPtr<SerializedScriptValue> m_serializedState;
```

Comment:
Have you condered storing the state on the HistoryItem? It seems to be what the spec and chromium implementation do.

Reply:
Done.